meson.build

```diff
@@ -2170,6 +2170,9 @@ simple_tests = []
 fuzzers = []
 simple_fuzzers = []
 catalogs = []
+modules = [] # nss, pam, and other plugins
```

Comment:
Since it creates shared libraries, shouldn't this be named as such?

Reply:
Currently, `libsystemd-shared`, `libsystemd`, or `libudev` are not built with this way... So, I'd like not to name the array shared_libraries or so.